ktlint-ruleset-standard/src/test/kotlin/com/pinterest/ktlint/ruleset/standard/rules/WrappingRuleTest.kt

```diff
@@ -1871,6 +1871,303 @@ internal class WrappingRuleTest {
             .hasLintViolation(2, 10, "A newline was expected before 'Bar'")
             .isFormattedAs(formattedCode)
     }
+
+    @Test
+    fun `Issue 1078 - Given a multiline semi separated var initialisation then wrap each expression to a new line`() {
+        val code =
+            """
+            fun f1() {
+                val a = 3; val b = 2
+                val key: String = ""; val lambda: () -> Unit = {  }
+            }
+            """.trimIndent()
+        val formattedCode =
+            """
+            fun f1() {
+                val a = 3;
+                val b = 2
+                val key: String = "";
+                val lambda: () -> Unit = {  }
+            }
+            """.trimIndent()
+        wrappingRuleAssertThat(code)
+            .hasLintViolations(
+                LintViolation(2, 15, "Missing newline after \";\""),
+                LintViolation(3, 26, "Missing newline after \";\""),
+            )
+            .isFormattedAs(formattedCode)
```

Comment:
Keep lines like this together on one line.

Reply:
Updated. BTW @paul-dingemans do we also have some rules to enforce this formatting? Or is it related to https://github.com/pinterest/ktlint/issues/1953 Or should we create a separate issue for that?

Reply:
The idea was to enforce this with #1953.